lib/plugins/aws/package/lib/saveServiceState.js

```diff
@@ -19,6 +19,8 @@ module.exports = {
       package: {
         individually: this.serverless.service.package.individually,
         artifactDirectoryName: this.serverless.service.package.artifactDirectoryName,
+        artifact: this.serverless.service.package.artifact,
+        artifacts: this.serverless.service.package.artifacts,
```

Comment:
I added `artifacts` for safety. It will contain the function artifacts, but it is not really needed, because `extendedValidate` will separately loop through the function artifacts by using the `getFunctionArtifact()` function.
Feel free to remove it - the fix works fine without it.